preview/MsixCore/MsixCoreInstaller/PrepareDevirtualizedRegistry.cpp

```diff
@@ -25,9 +25,11 @@ HRESULT PrepareDevirtualizedRegistry::ExtractRegistry(bool remove)
 {
     std::wstring registryFilePath = m_msixRequest->GetPackageDirectoryPath() + registryDatFile;

```

Comment:
>registryFilePath [](start = 17, length = 16)

we may want to make a copy of this and store it somewhere so that we preserve the original copy.

Reply:
Copied registry.dat file to registryFileCopy.dat and passed copy file to other handlers.